bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -1572,9 +1572,8 @@
 	</channel-type>
 
 	<channel-type id="controlSignalCirculatingPump" advanced="true">
-		<item-type>Number:Energy</item-type>
+		<item-type>Number:Dimensionless</item-type>
 		<label>Control Signal Circulating Pump</label>
```

Comment:
Should this label be updated as well, like you did in the README? The README version is too long for a label, so a description could also help.